Library/Homebrew/cask/installer.rb

```diff
@@ -93,6 +93,7 @@ def install
 
         raise CaskAlreadyInstalledError, @cask
       end
+      predecessor = (reinstall? && @cask.installed?) ? @cask : nil
```

Comment:
```suggestion
      predecessor = @cask if reinstall? && @cask.installed?
```
produces the same result and is more readable IMO